drivers/sensors/Kconfig

```diff
@@ -2159,51 +2159,6 @@ config TMP112_I2C_FREQUENCY
 	int "TMP112 I2C frequency"
 	default 400000
 
-config TMP112_I2C_ADDR_1
```

Comment:
why add in the first patch and remove here, let's modify the first patch instead